recipes/tensorflow-lite/all/patches/dependencies_2_6.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
-index f5cc9b97924..4edc693dd7d 100644
+index f5cc9b97924..b4876e166ef 100644
 --- a/tensorflow/lite/CMakeLists.txt
 +++ b/tensorflow/lite/CMakeLists.txt
 @@ -50,14 +50,8 @@ if(NOT TENSORFLOW_SOURCE_DIR)
```

Comment:
are you able to drop this section? I found it was not required in my PR

Reply:
Nice catch! Done :)